packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -343,6 +349,18 @@ export function shouldSetTextContent(type: string, props: Props): boolean {
 }
 
 export function getCurrentEventPriority(): * {
+  const currentEventPriority = fabricGetCurrentEventPriority();
```

Comment:
What if fabricGetCurrentEventPriority is not defined or nativeFabricUIManager doesn't exist?